net-dns/yaddns/yaddns-1.1.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="A small dynamic DNS client"
+DESCRIPTION="Yaddns is a small dynamic dns client with multiple servers \
+support and a high flexibility."
+HOMEPAGE="https://yaddns.github.io/"
+COPYRIGHT="2015 Anthony Viallard and Raphael Huck"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/yaddns/yaddns/archive/yaddns-$portVersion.tar.gz"
+CHECKSUM_SHA256="23b527b7f71e1449746f1e39cbc3a074d58bd261a33399fb4e2d90eb63149059"
+SOURCE_DIR="yaddns-yaddns-$portVersion"
+PATCHES="yaddns-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]
+then
+	commandBinDir=$prefix/bin
+	commandSuffix=
+fi
+
+PROVIDES="
+	yaddns$secondaryArchSuffix = $portVersion
+	cmd:yaddns$commandSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcrypto$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autom4te
+	cmd:automake
+	cmd:autoreconf
+	cmd:g++$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:makeinfo
+	"
+
+USER_SETTINGS_FILES="
+	settings/yaddns.conf
+	"
+
+defineDebugInfoPackage yaddns$secondaryArchSuffix \
+	$commandBinDir/yaddns
+
+PATCH()
+{
+	sed -i -e "s|/etc\(/yaddns\.conf\)|`finddir B_SYSTEM_SETTINGS_DIRECTORY`\1|;" \
+		doc/yaddns.1
+}
+
+BUILD()
+{
+	./autogen.sh
+	CPPFLAGS="-D_BSD_SOURCE" \
+		LIBS="-Wl,--as-needed -lbnetapi -lbsd -lnetwork" \
```

Comment:
Thanks for the update!
May I ask a tiny style change?
```diff
 	CPPFLAGS="-D_BSD_SOURCE" \
-		LIBS="-Wl,--as-needed -lbnetapi -lbsd -lnetwork" \
+	LIBS="-Wl,--as-needed -lbnetapi -lbsd -lnetwork" \
 		runConfigure --omit-dirs binDir ./configure --bindir $commandBinDir
```
(It's just the indentation in the second line.)

Thanks!